src/adapters/supabase/helpers/user.ts

```diff
@@ -17,10 +17,10 @@ export class User extends Super {
     if ((error && !data) || !data.wallets?.address) {
       this.context.logger.error("No wallet address found", { userId, issueNumber });
       if (this.context.config.startRequiresWallet) {
-        await addCommentToIssue(this.context, "```diff\n! Please set your wallet address with the /wallet command first and try again.\n```");
+        await addCommentToIssue(this.context, this.context.config.emptyWalletText);
         throw new Error("No wallet address found");
       } else {
-        await addCommentToIssue(this.context, "```diff\n# Please set your wallet address with the /wallet command in order to be eligible for rewards.\n```");
```

Comment:
@rndquu There was a slight variance here because this comment is intended to be passive while the comment above is intended to be assertive.

Is the plan to write one response that will accurately represent both outcomes here or should we extend the config item?

`config.customStrings.walletRequired`
`config.customStrings.walletSuggestion`

Then any future configurable strings can easily be inserted here?

Reply:
https://github.com/ubiquibot/plugin-template/issues/19